### PR TITLE
Delete leftover plots

### LINKED ISSUE #19: While testing, killing a simulation leaves behind the generated plots
## Issue
Right now, while testing, the code waits only for 10 minutes for a simulation to complete (while tweeting, this time limit is 20 minutes) to make the tests quicker but sometimes the simulation is canceled midway and the leftover generated plots are never deleted.

## Fix
As I don't want the tests to go on for even longer (they already run for about an hour right now), it would be good to programmatically delete these images rather than increasing the time out.

### PR DESCRIPTION
**Pull request by Saransh-cpp**
Fixes #19 

Added code to delete all the PNGs if they are leftover while testing.

### CODE REVIEW

File: bot/plotting/comparison_generator.py [hunk 1]
```diff
@@ -79,9 +79,23 @@ def comparison_generator(
...
+                param_value = parameter_value_generator(
+                    chemistry, param_to_vary
+                )
+                if (
+                    param_to_vary == "Electrode height [m]"
+                    or param_to_vary == "Electrode width [m]"
+                ):     # pragma: no cover
+                    if (
+                        param_value == 0
+                        or param_value < 0
```
Comment (brosaplanella) [inline bot/plotting/comparison_generator.py:92]: Is there any particular reason to not use `<=`?

**Review by brosaplanella — APPROVED** [review]
Looks good, thanks Saransh!

File: bot/plotting/comparison_generator.py (hunk 1 above)
Comment (Saransh-cpp) [reply bot/plotting/comparison_generator.py:92]: No particular reason, a silly mistake:)

Comment (brosaplanella) [reply bot/plotting/comparison_generator.py:92]: Technically it is not a mistake haha

**Pull request merged**